src/tests/remapDataNode.test.js

```diff
@@ -0,0 +1,285 @@
+require("dotenv").config();
+const RemapDataNode = require("../nodes/remapDataNode");
+
+const nodeSchema = {
+  id: "1",
+  name: "rightSchema",
+  next: "2",
+  lane_id: "any",
+  type: "systemTask",
+  category: "remapData",
+  parameters: {
+    input: {
+      data: [{ "field1": 1 }],
+      dictionary: { "field": "field1" }
+    },
+  },
+}
+
+const executionDataExample = {
+  data: [
+    {
+      id: 1,
+      date: "01/01/2005",
+      user: { name: "test" }
+    }
+  ],
+  dictionary: {
+    identification: "id",
+    name: "user.name"
```

Comment:
adicionar um teste com pelo menos +1 nível de aninhamento

Reply:
Trata sim, o teste que cobre é o `"should work with nested object on dictionary"` usando o `nodeExampleNestedObject`